test/createAzureResource.test.ts

```diff
@@ -112,6 +119,27 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         assert.equal(createdApp.scmType, constants.ScmType.None, `Web App scmType's property value should be ${constants.ScmType.None} rather than ${createdApp.scmType}.`);
     });
 
+    test('Create a new windows Web app and deployment to Web App', async () => {
+        const resourceGroupName: string = getRandomHexString();
+        const webAppName: string = getRandomHexString();
+        const AppServicePlan: string = getRandomHexString();
+        const inputs: string[] = [];
+        await fse.ensureDir(testFolderPath);
+        resourceGroupsToDelete.push(resourceGroupName);
+        const uri: string = 'https://codeload.github.com/Azure-Samples/nodejs-docs-hello-world/zip/master';
```

Comment:
Rather than downloading and unzipping a file, I'd rather you just put the project in our test folder and deploy it from there.  If it's small, it's fine to include it in the repo.

From there, if you want, we can have a .zip file and a folder to deploy since they are two different scenarios.